templates/project.html

```diff
@@ -50,18 +58,25 @@
 
 {% else %}
 .question-group {
-  margin: 30px 0 20px 0;
+  margin: 10px 0 20px 0;
 }
 
 h2.question-column-title {
 }
 
 h3.question-group-title {
-  color: #777;
+  color: black;
+  font-weight: bold;
   padding: 16px 18px;
   margin: 0;
 }
 
+.question-row-container {
```

Comment:
This should just be on .question since `question-row-container` is on exactly the same elements as `question`.

Reply:
same.